Sources/SwiftFormat/Core/RuleMask.swift

```diff
@@ -111,10 +111,11 @@ enum IgnoreDirective: CustomStringConvertible {
     }
   }
 
-  /// Regex pattern to match an ignore comment. This pattern supports 0 or more comma delimited rule
-  /// names. The rule name(s), when present, are in capture group #3.
-  fileprivate var pattern: String {
-    return #"^\s*\/\/\s*"# + description + #"((:\s+(([A-z0-9]+[,\s]*)+))?$|\s+$)"#
+  /// Regex pattern to match an ignore directive comment.
+  /// - Capture group #1 captures the rule names if `":"` is present.
+  fileprivate func makeRegex() -> Regex<(Substring, Substring?)> {
+    let pattern = #"^\s*\/\/\s*"# + description + #"(?:\s*:\s*(.+))?$"#
+    return try! Regex(pattern)
```

Comment:
Could you add a comment that we can’t use regex literals here because Windows didn’t have support for regex literals until Swift 5.10 (at least I think it was 5.10)? Just so we don’t go back and wonder why we aren’t using regex literals  here.

Reply:
Sure, I've added it.